docs/src/api_comparison.md

```diff
@@ -88,9 +88,9 @@ welcome.
 | **Segmentation**                      |                                           |                                   |                                           |
 | Connected components                  | `label_components`                        | `label`                           | `bwconncomp`, `bwlabel`                   |
 | Foreground/background                 |                                           | `active_contour`                  | `activecontour`                           |
-| Clustering                            |                                           | `quickshift`                      |                                           |
+| Clustering                            | ` kmeans`, `fuzzy_cmeans`                 | `quickshift`                      |                                           |
 | Marker segmentation                   |                                           | `random_walker`                   | `imsegfmm`                                |
```

Comment:
Perhaps add `seeded_region_growing` to the Julia column?

Reply:
Same doubt as above.

Reply:
To `Marker segmentation`